src/components/10-atoms/input-text/index.js

```diff
@@ -272,23 +280,23 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
 
         // eslint-disable-next-line no-restricted-globals
         if (!isNaN(valueDecimalsOnly)) {
-          this.invalid = false;
+          this.invalidFormat = false;
           return this.currencyFormatter.format(valueDecimalsOnly);
         }
       }
 
-      this.invalid = value.length > 0;
+      this.invalidFormat = value.length > 0;
     }
 
     return value;
   }
 
   firstUpdated() {
-    const { defaultValue, isReact, value } = this;
+    const { defaultValue, isReact } = this;
     this._setNativeInput();
 
-    if (isReact) {
-      this.nativeInput.value = defaultValue || value;
+    if (isReact && defaultValue) {
+      this.nativeInput.value = this._formatCurrency(defaultValue);
```

Comment:
You has added `defaultValue` to the if. And removed value. Isnt it possible that defaultValue is false and value is truthy?

Reply:
Yes thats true but `value` is set during render and therefore there is no need to set it again.